app/Language/en-ca/Legal.php

```diff
@@ -0,0 +1,85 @@
+- Use the ® trade-mark registration symbol when you use the CodeIgniter registered trade-mark.
+- Use capitalization and bold or italic fonts to distinguish the CodeIgniter trade-mark from the text around it.
+- Place a conspicuous legend at least once on webpages using the CodeIgniter trade-mark and 
+	on the title page of any documentation or sales literature bearing the CodeIgniter trade-mark. 
+	An example of an acceptable legend is: "The CodeIgniter mark is owned and may be registered by BCIT."
+- Place a disclaimer conspicuously at least once on webpages using the CodeIgniter 
+	trade-mark and on the title page of any documentation or sales literature bearing 
+	the CodeIgniter trade-mark. An example of an acceptable disclaimer is: "\_________ is an 
+	independent \_________ and has not been authorized, sponsored, or otherwise approved by BCIT."  
+	[In each case, fill the blank space with the name and type of your product.]
+- Do not change the spelling, make other changes to the CodeIgniter trade-mark, 
+	or combine the CodeIgniter trade-mark with any other trade-mark, term, or designation.
+- Use the CodeIgniter trade-mark only in connection with goods and services directly 
+	related to CodeIgniter or services associated with that trade-mark.
+- Do not use the CodeIgniter trade-mark or any variation of it in your domain name or URL.
+- Do not use the CodeIgniter trade-mark for your own products and services that are 
+	related to web publishing, programming frameworks, or related software.
+- Do not claim ownership rights in the CodeIgniter trade-mark.
+EOS;
+
+$thirdParty = <<<'EOS'
+Mention of third party companies and products on this web site is for informational 
+purposes only and constitutes neither an endorsement nor a recommendation.
+EOS;
+
+$externalLinks = <<<'EOS'
+BCIT and the CodeIgniter Foundation make no representations whatsoever about any 
+other websites which you may access from here. When you access an external website, 
+please understand that it is independent from the BCIT website and from this 
+website, and that we has no control over the content on that website. A link to 
+a non-BCIT we site does not mean that BCIT or the CodeIgniter Foundation endorse 
+or accept any responsibility for the content or use of such website.
+EOS;
+
+return [
+	'title'				 => 'The Fine Print',
+	'marksTitle'		 => 'Trade-marks',
+	'marksText'			 => $trademarks,
+	'marksLink'			 => '[several versions of the CodeIgniter logo](/data/ci-logo.zip)',
+	'logoTitle'			 => 'Privacy Policy',
+	'logoText'			 => $logoGuidelines,
+	'logoLink'			 => '[several versions of the CodeIgniter 4 logo](/data/ci4-logo.zip)',
+	'thirdPartyTitle'	 => 'Website data',
+	'thirdPartyText'	 => $thirdParty,
+	'externalLinksTitle' => 'Links to Other Web Sites',
+	'externalLinksText'	 => $externalLinks,
+	'endText'			 => 'We invite you to read BCIT\'s [main privacy page](http://www.bcit.ca/privacy/)'
+];
```

Comment:
I don't see any text here that differs fro the "en" verison.

Reply:
I have replaced "trademark" with "trade-mark" as per canadian english standards.